packages/exhibitor/extra/dcos-exhibitor.service

```diff
@@ -10,6 +10,9 @@ RestartSec=5
 # Run in new mount namespace to create custom resolv.conf.
 MountFlags=private
 RuntimeDirectory=dcos_exhibitor
+# On some systems /tmp is mounted as noexec. Make zookeeper write its JNA
+# libraries to a path we control instead. See DCOS-11056
+Environment=SERVER_JVMFLAGS=-Djna.tmpdir=/var/lib/dcos/exhibitor/tmp
```

Comment:
Wouldn't it make more sense to write this value into the existing `/opt/mesosphere/etc/exhibitor` env file?